conf/evolutions/default/19.sql

```diff
@@ -0,0 +1,5 @@
+# --- !Ups
+ALTER TABLE "user" RENAME TO sidewalk_user;
```

Comment:
Would this work since user is a table (outside of sidewalk) in itself? Shouldn't this be sidewalk.user?

Reply:
Both `sidewalk.user` and quoted `"user"` work!